debian/changelog

```diff
@@ -1,3 +1,12 @@
+libfabric (1.18.0-1) unstable; urgency=medium
+  [ Jerome Soumagne ]
+  * Update to 1.18.0
+  * Add libnuma-dev dependency
+  * Clean up spec file and disable unused / deprecated providers
```

Comment:
`spec` file?  Do you mean `control` file, or perhaps the `rules` file?

Reply:
well for control/rules files it really just aligned with the spec file, I should update that log message to reflect that.